samples/graph-meeting-notification/nodejs/helper/certificate-helper.js

```diff
@@ -0,0 +1,130 @@
+// <copyright file="certificate-helper-bot.js" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+const fs = require('fs');
+const path = require('path');
+const os = require('os');
+const crypto = require('crypto');
+const pem = require('pem');
+/**
+ * Configures path to openssl.exe if needed
+ */
+function ensureOpenSsl() {
+  if (os.platform() === 'win32') {
+    const pathOpenSSL = process.env.OPENSSL_CONF
+      ? process.env.OPENSSL_CONF.replace('.cfg', '.exe')
+      : 'C:/Program Files/OpenSSL-Win64/bin/openssl.exe';
```

Comment:
Can we get this parameter as part of .env file?

Reply:
Added in .env file.